dygraph/mobilenet_v1/utils/utility.py

```diff
@@ -0,0 +1,576 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
too many Irrelevant codes in here, I think should create an independent dygraph utility file